### PR TITLE
Reskin Fix #1806 - Add * for required fields

### LINKED ISSUE #1806: Reskin&oldSkin: Asterisk sign needed to indicate that  Address Type and street are mandatory fields
Asterisk sign needed to indicate that  **Address Type** and **Street** are mandatory fields in the client creation form at the address category. 
mf

### PR DESCRIPTION
**Pull request by gkrishnan724**
required

### CODE REVIEW

**Comment by mbj36** [comment]
@botraunak Can you guide @gkrishnan724 in this issue as you have solved earlier this one. 
This is not the correct fix

File: app/views/clients/createclient.html [hunk 1]
```diff
@@ -265,7 +265,7 @@
                      <div ng-repeat="addr in addressArray">
                         <div class="form-group" >
                            <div class="control-label col-sm-2">
-                              <label>{{ 'label.input.addressType' | translate }}</label>
+                              <label>{{ 'label.input.addressType' | translate }}<span class="required">*</span></label>
                            </div>
                            <div class="col-sm-3"  >
                               <select chosen="addressTypes"  ng-model="addressArray[$index].addressTypeId"  class="form-control"
```
Comment (botraunak) [inline app/views/clients/createclient.html]: Need to add required and late-validate attributes, just adding * won't fix this.

File: app/views/clients/createclient.html [hunk 2]
```diff
@@ -276,7 +276,7 @@
                         </div>
```
Comment (botraunak) [inline app/views/clients/createclient.html]: Add form-validate tag to show the validation results.

File: app/views/clients/createclient.html [hunk 3]
```diff
@@ -276,7 +276,7 @@
...
                         <div class="form-group" ng-show="street">
                            <div class="control-label col-sm-2">
-                              <label>{{ 'label.input.street' | translate }}</label>
+                              <label>{{ 'label.input.street' | translate }}<span class="required">*</span></label>
                            </div>
                            <div class="col-sm-3">
                               <input type="text" name="street"  ng-model="addressArray[$index].street"  class="form-control"/>
```
Comment (botraunak) [inline app/views/clients/createclient.html]: Add required and late-validate and form-validate tags to fix this appropriately.

**Comment by gkrishnan724** [comment]
@botraunak @mbj36

**Comment by mbj36** [comment]
Thanks @gkrishnan724

**Pull request merged**